lib/experimental/webfonts.php

```diff
@@ -76,25 +74,68 @@ function wp_register_webfonts( array $webfonts = array() ) {
  * Example of how to register Source Serif Pro font with font-weight range of 200-900:
  *
  * If the font file is contained within the theme:
- * ```
+ *
+ * <code>
  * wp_register_webfont(
  *      array(
  *          'provider'    => 'local',
- *          'font_family' => 'Source Serif Pro',
- *          'font_weight' => '200 900',
- *          'font_style'  => 'normal',
+ *          'font-family' => 'Source Serif Pro',
+ *          'font-weight' => '200 900',
+ *          'font-style'  => 'normal',
  *          'src'         => get_theme_file_uri( 'assets/fonts/source-serif-pro/SourceSerif4Variable-Roman.ttf.woff2' ),
  *      )
  * );
- * ```
+ * </code>
  *
  * @since 6.0.0
  *
  * @param array $webfont Webfont to be registered.
- *                       See {@see WP_Webfonts_Registry::register()} for a list of supported arguments.
  */
 function wp_register_webfont( array $webfont ) {
-	wp_webfonts()->register_font( $webfont );
+	wp_webfonts()->register_webfont( $webfont );
+}
+
+/**
+ * Enqueues a collection of font families.
+ *
+ * Example of how to enqueue Source Serif Pro and Roboto font families, both registered beforehand.
+ *
+ * <code>
+ * wp_enqueue_webfonts(
+ *  'Roboto',
+ *  'Sans Serif Pro'
+ * );
+ * </code>
+ *
+ * Font families should be enqueued from the `init` hook or later.
+ *
+ * @since 6.0.0
+ *
+ * @param string[] $webfonts Font families to be enqueued.
+ */
+function wp_enqueue_webfonts( $webfonts ) {
```

Comment:
Type hint with `array`.

```suggestion
function wp_enqueue_webfonts( array $webfonts ) {
```

Reply:
Resolved in https://github.com/WordPress/gutenberg/pull/39559/commits/27e7da71a053c7e21c91d83935fbb1fd460f376b.